tmt/schemas/stories.yaml

```
@@ -0,0 +1,63 @@
+---
+
+#
+# JSON Schema definition for tmt `Stories`
+#
+# https://tmt.readthedocs.io/en/stable/spec/stories.html
+#
+
+$id: /schemas/stories
```

```suggestion
$id: /schemas/story
```

I guess the id should be updated as well, right?

---

yeah